### DISCUSSION TITLE
Update static canvas infinitely

### DISCUSSION THREAD

**Discussion by Johnson-LYS** [pixijs/pixijs#7701] [Q&A]
### Expected Behavior
Don't update canvas when it is static.
### Current Behavior
Undate(re-render) every frame even if canvas is empty.
![image](https://user-images.githubusercontent.com/19748456/129171086-d14c9356-0825-4240-ad50-086be469b0c7.png)

### Environment

- **`pixi.js` version**: _e.g. 4.7.1_
- **Browser & Version**: _e.g. Chrome 67_
- **OS & Version**: _e.g. Ubuntu 18.04_
- **Running Example**: _url_

**Comment by eXponenta** [comment]
This is impossible.
Pixi not has and NEVER will have a diff check mechanism.

You can run animation inside shader and pixi MUST call render for it, but there are not way track changing in shader internally.

You can only disable internal render loop by app.stop(), and runs it when required manually
